content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -84,6 +102,8 @@ resources from storage, each provider that matches the stored data attempts in o
 data. If no provider can read the stored data due to a mismatch in format or secret key, an error
 is returned which prevents clients from accessing that resource.
 
+`EncryptionConfiguration` also supports the use of wildcards to specify the resources that should be encrypted. Use '`*.<group>`' to encrypt all resources within a group (for eg '`*.apps`' in above example) or '`*.*`' to encrypt all resources. '`*.`' can be used to encrypt all resource in the core group.  '`*.*`' will encrypt all resources, even custom resources that are added after API server start. Use of wildcards that overlap within the same resource list or across multiple entries are not allowed since part of the configuration would be ineffective. Resource lists are processed in order, with earlier lists taking precedence.
```

Comment:
```suggestion
`EncryptionConfiguration` supports the use of wildcards to specify the resources that should be encrypted. Use '`*.<group>`' to encrypt all resources within a group (for eg '`*.apps`' in above example) or '`*.*`' to encrypt all resources. '`*.`' can be used to encrypt all resource in the core group.  '`*.*`' will encrypt all resources, even custom resources that are added after API server start. 
NOTE: Use of wildcards that overlap within the same resource list or across multiple entries are not allowed since part of the configuration would be ineffective. The `resources` list's processing order and precedence are determined by the order it's listed in the configuration.
```

Reply:
Please wrap the long line appropriately as we did for other paragraphs.